.idems_app/deployments/plh/skins/debug.ts

```diff
@@ -0,0 +1,18 @@
+import { IAppSkin } from "data-models";
+
+const debug: IAppSkin = {
+  name: "debug",
+  appConfig: {
```

Comment:
I've refactored this skin to be in line with how I saw skins as working: namely, rather than including the full default app config (from `getDefaultAppConfig()`) with some changes applied, the skin includes a partial `appConfig` containing only the values that differ from the default `appConfig`.
Applying the skin's config values is is handled by the `appConfigService.updateAppConfig()` method, which deep merges the `appConfig` from the deployment with the the skin's `appConfig`. Through the power of the deep merge, I was considering a skin's `appConfig` to be a minimal set of specific _overrides_, not as a full appConfig with some alterations like that of a deployment. 
See other comment for an example of why it makes sense to me to keep skins' `appConfig` as a minimal set of overrides.
Although there may well be downsides or issues with this approach that I haven't considered

Reply:
Yeah that makes sense, easier to author that way and I remember we already have methods to support deep-merging of objects so good to use. As part of tests at some point it would be good to also include function overrides just to make sure they are also handled correctly (skin config will be converted to string, so can't remember if the deep merge will also re-parse)